test/unit/simulate.js

```diff
@@ -1,4 +1,8 @@
 (function() {
+var key = jQuery.simulate.keyCode,
+	clickOptions = [ "ctrlKey", "altKey", "shiftKey", "metaKey" ],
```

Comment:
Separate variables for clickOptions and keyOptions might be unnecessary since they're currently the same, but thought it would be nice for any potential future options testing if they were split up.